brokenScript.js

```diff
@@ -0,0 +1,114 @@
+$(document).ready(function(){
+  ticTacToe = {
+    XorO:['url("LETTERO.png")',' url("LETTERX.jpg")'],
+    clickCount: 1,
+    squareClicks: 0,
+    chooseWhich: function(){
+      if((ticTacToe.clickCount/2)==1){
+        ticTacToe.clickCount--;
+        $('.whosTurn').css('background-image',ticTacToe.XorO[ticTacToe.clickCount-1]);
+      }
+      else{
+        ticTacToe.clickCount++;
+        $('.whosTurn').css('background-image',ticTacToe.XorO[ticTacToe.clickCount-1]);
+      }
+    },
+    closeChooser: function(){
+      if (ticTacToe.squareClicks == 1){
+        $('.whosTurn').off('click');
+
+      }
+    },
+    whichClick: function(){
+      $(this).off('click');
+      if(ticTacToe.clickCount==2){
+        $(this).css('background-image',ticTacToe.XorO[ticTacToe.clickCount-1]);
+        ticTacToe.squareClicks++;
+        ticTacToe.getWinner();
+        ticTacToe.closeChooser();
+        console.log(ticTacToe.clickCount)
+        ticTacToe.clickCount--;
+        $('.whosTurn').css('background-image',ticTacToe.XorO[ticTacToe.clickCount-1]);
+        console.log(ticTacToe.clickCount);
+      }
+      else if (ticTacToe.clickCount==1){
+        console.log("click count isn't 2")
+        console.log(ticTacToe.XorO[ticTacToe.clickCount-1]);
+        $(this).css('background-image',ticTacToe.XorO[ticTacToe.clickCount-1]);
+        ticTacToe.squareClicks++;
+        ticTacToe.closeChooser();
+        ticTacToe.getWinner();
+        console.log(ticTacToe.clickCount);
+        ticTacToe.clickCount++;
+        $('.whosTurn').css('background-image',ticTacToe.XorO[ticTacToe.clickCount-1]);
+        console.log(ticTacToe.clickCount);
+      }
+
+    },
+    declareWinner: function(){
+      if (ticTacToe.clickCount == 1){
+        alert('O WINS!');
+        ticTacToe.resetGame()
+        }
+      else if (ticTacToe.clickCount == 2) {
+        alert('X WINS!');
+        ticTacToe.resetGame()
+      }
+    },
+    getWinner: function(){
+
```

Comment:
Consider adding an array of possible square number combinations,  which you iterate through in your `checkWinner` functions to check against an array of player moves